index.js

```diff
@@ -65,7 +65,14 @@ BroccoliMergeTrees.prototype.build = function() {
 
   var applyPatchStart = new Date();
 
-  this._applyPatch(patch);
+  try {
+    this._applyPatch(patch);
+  } catch(e) {
+    // Whatever the failure, start again and do a complete build next time
+    this._currentTree = FSTree.fromPaths([]);
```

Comment:
Don't we also need to actually clear the output directory with `rimraf`? If we pretend it's empty but it's not, then the stale files will be in the way the next time we build.